packages/gpiod/gpiod.0.7/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis:
+  "A wrapper around the C libgpiod library for GPIO on recent (>4.8) Linux kernels"
+maintainer: ["Blake Loring <blake@parsed.uk>"]
+authors: ["Blake Loring"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/jawline/ocamlGpiod/"
+bug-reports: "https://github.com/jawline/ocamlGpiod/"
+depends: [
+  "dune" {>= "2.8"}
+  "re"
+  "core"
+  "ocaml" {>= "4.08.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/jawline/ocamlGpiod.git"
+depexts: [
+  "libgpiod-dev"
+]
+available: [ os = "linux" & os-family = "debian"  ]
```

Comment:
Is gpiod debian specific or we could drop this line and add the relevant depexts?

Reply:
Ah I really should - sorry I'm still getting the hang of depexts. I'll do some citation chasing for other distributions when I have a moment.

Reply:
Not all distributions include it, I've marked all those I could fine with packages but I'm having some trouble eliminating oraclelinux from the build. Is there any table of all distributions os, os-distribution and os-family data anywhere? I can't install opam in docker on oraclelinux so it's not easy for me to do an adhoc check.